services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/core/errors.py

```diff
@@ -9,13 +11,11 @@ class ConfigurationError(ResourceUsageTrackerRuntimeError):
     msg_template: str = "Application misconfiguration: {msg}"
 
 
-class CreateServiceRunError(ResourceUsageTrackerRuntimeError):
-    msg_template: str = "Error during creation of new service run record in DB: {msg}"
+class MyHTTPException(HTTPException):
```

Comment:
I do not see any advantages on this artificial exception.
If you do not want to use custom exceptions you should just use `HTTPExceptions`.
Besides, I think exceptions is part of the api layers. The same way you are separating functions between controller-servcies-repo layers it makes sense to me to have a way to distinguish errors in those. I understand it can be combersome if you just map one-to-one but the handlers functions as `my_http_exception_handler` can be used to do the mappings properly.